src/test/java/com/sendgrid/helpers/MailTest.java

```diff
@@ -170,7 +170,7 @@ public void testKitchenSink() throws IOException {
     TrackingSettings trackingSettings = new TrackingSettings();
     ClickTrackingSetting clickTrackingSetting = new ClickTrackingSetting();
     clickTrackingSetting.setEnable(true);
-    clickTrackingSetting.setEnableText(true);
+    clickTrackingSetting.setEnableText(false);
```

Comment:
@thinkingserious This seems like more of a problem with the setEnable function? Why are we doing 
```clickTrackingSetting.setEnable(true);```
and then
```clickTrackingSetting.setEnableText(true);```

Isn't that redundant? 
In [Setting](https://github.com/sendgrid/sendgrid-java/blob/master/src/main/java/com/sendgrid/helpers/mail/objects/Setting.java) we only return enable - I didn't see a place where we return enableText - is that for the "pretty" print? If so, can we modify the class to return enable instead of enableText in that case?

Reply:
The real change was removing `@JsonInclude(Include.NON_DEFAULT)` from ClickTrackingSetting.java to solve issue #181. I think the test was changed to demonstrate that the change worked.